moban/filters/text.py

```diff
@@ -1,6 +1,8 @@
 import re
+from moban.extensions import JinjaFilter
 
 
+@JinjaFilter("split_length")
```

Comment:
maybe for a future release, it would be nice if `@JinjaFilter()`, without args, can auto-detect the wrapped function name and use it.

Reply:
I havent thought of it till now. Yes, can use the function name for the filter, hence DRY is achieved